controllers/account/account_controller.go

```diff
@@ -117,6 +121,33 @@ func (r *AccountReconciler) Reconcile(ctx context.Context, request ctrl.Request)
 		return reconcile.Result{}, err
 	}
 
+	// Read compliance tags feature flag
```

Comment:
Let's break this whole chunk of parsing logic out into it's own private function, something like `func (r *AccountReconciler) generateAccountTags()`